en/sql/schema/view_stmt.rst

```diff
@@ -294,15 +294,55 @@ You can drop a query defined in the query specification by using the **DROP QUER
 COMMENT Clause
 --------------
 
-You can change a view's comment with **COMMENT** clause of **ALTER VIEW** syntax.
+You can change a view's comment and column comments with **COMMENT** clause of **ALTER VIEW** syntax.
 
 ::
 
-    ALTER [VIEW | VCLASS] view_name COMMENT [=] 'view_comment';
+    ALTER [VIEW | VCLASS] view_name
+    COMMENT [=] 'view_comment_string';
+    COMMENT ON {COLUMN | CLASS ATTRIBUTE} <column_comment_definition> [, <column_comment_definition>] ;
+
+        <column_comment_definition> ::= column_name [=] 'column_comment_string'
+
+*   *view_name*: specifies the name of a view to be modified.
+*   *column_name*: specifies the name of a column to be modified.
+*   *view_comment_string*: specifies a view's comment.
+*   *column_comment_string*: specifies a column's comment.
```

Comment:
specifies --> Specifies
(Since the other part of the manual use capital letter after ':', I think is better in there, we also capitalise the word 'specifies')

Reply:
Thank you. Fixed in '1989cf0' commit.